src/pages/Admin/Venue/Users/VenueUsersReport.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useEffect, useCallback, useState } from "react";
+import { useWorldUsers } from "hooks/users";
+import "./VenueUsersReport.scss";
+import Table from "react-bootstrap/Table";
+import Form from "react-bootstrap/Form";
+import { downloadGeneratedCSVFile } from "utils/csv";
+import { User } from "types/User";
+import { WithId } from "utils/id";
+
+const VenueUsersReport: React.FC = () => {
+  const { worldUsers } = useWorldUsers();
+
+  const [headers, setHeaders] = useState<Array<string>>([]);
+  const [headersSelected, setHeadersSelected] = useState<Array<string>>([]);
+  const [customFieldValue, setCustomFieldValue] = useState("");
+
+  const userIsComplete = useCallback((user: WithId<User>) => {
+    return user.partyName && user.pictureUrl;
+  }, []);
```

Comment:
This function is already defined in a different file somewhere. Why are we not just importing + calling it here. We definitely don't want to be duplicating logic like this.

Reply:
The input of the global one is different, it gets `user profile` so, that's why I could not use that one. am I right?

Reply:
I’m pretty sure they’re exactly the same in both cases here and so you should be able to use the existing one.